### PR TITLE
12.0 fix 780

### LINKED ISSUE #780: [12.0] Port pos_debt_notebook
I am creating this issue to discuss update of the pos_debt_notebook module.

### PR DESCRIPTION
**Pull request by carlosrve**
Preliminary PR (fixes #780 ) to update pos_debt_notebook to v12. We are doing some functional tests. It is installing OK and everything is working so far.

### CODE REVIEW

**Comment by carlosrve** [comment]
The travis configuration is for v11 on the repo, so the tests will not succeed. I suppose that you are changing this on another PR/commit, right?

**Comment by yelizariev** [comment]
Yes, I'll take care about CI tools

**Comment by yelizariev** [comment]
I've merged 11.0 to 12.0 branch, so you need to rebase or resolve conflicts via github UI

**Comment by carlosrve** [comment]
I rebased the PR on the current v12 branch.

**Comment by GabbasovDinar** [comment]
@yelizariev It has been solved here: https://github.com/it-projects-llc/pos-addons/commit/29a88823eddf2715adb74cdbd608c7e0191730b7 we can close the PR

File: pos_debt_notebook/views/pos_credit_update.xml [hunk 1]
```diff
@@ -67,13 +67,13 @@
                     <field name="partner_id"/>
                     <field name="user_id"/>
                     <group expand="1" string="Group By">
-                        <filter string="Salesperson" name="User" context="{'group_by':'user_id'}"/>
-                        <filter string="Customer" context="{'group_by':'partner_id'}"/>
+                        <filter string="Salesperson" name="user" context="{'group_by':'user_id'}"/>
```
Comment (Ramil-Mukhametzyanov) [inline pos_debt_notebook/views/pos_credit_update.xml:70]: This line was not moved to the commit https://github.com/it-projects-llc/pos-addons/commit/29a88823eddf2715adb74cdbd608c7e0191730b7.
What is the reason?

**Comment by KolushovAlexandr** [comment]
Thanks for contributing,
The module was ported to 12.0 here https://github.com/it-projects-llc/pos-addons/pull/815

**Pull request closed without merging**